### PR TITLE
UI integration

### PR DESCRIPTION
**Pull request by balthazzar**
https://github.com/webRunes/webGold-WRIO-App/issues/156
https://github.com/webRunes/Chess-WRIO-Game/issues/85
https://github.com/webRunes/Core-WRIO-App/issues/64

### CODE REVIEW

File: WRIO-InternetOS/js/components/CreateDomCenter.js [hunk 1]
```diff
@@ -164,25 +172,39 @@ class CreateDomCenter extends React.Component{
...
+           nocomments = true;
+           this.state.notDisplayCenter=true;
+            displayCore =  ( <iframe src={'http://core.'+process.env.DOMAIN+'/create'} style={ this.editIframeStyles }/>);
         }
 
         if (urlParams.edit) {
-            notDisplayCenter=true;
-            displayCore =  ( <iframe src={'http://core.'+process.env.DOMAIN+'/?edit=' + notDisplayCenter.href} style={ this.editIframeStyles }/>);
+            this.state.notDisplayCenter=true;
+            displayCore =  ( <iframe src={'http://core.'+process.env.DOMAIN+'/edit?article=' + window.location.host} style={ this.editIframeStyles }/>);
```
Comment (AlexeyAnshakov) [inline WRIO-InternetOS/js/components/CreateDomCenter.js:202]: use

> //core

not

> http://core

**Comment by AlexeyAnshakov** [comment]
@michbil please review

**Comment by michbil** [comment]
@AlexeyAnshakov Looks good to me

File: WRIO-InternetOS/js/components/CreateDomCenter.js [hunk 2]
```diff
@@ -164,25 +172,39 @@ class CreateDomCenter extends React.Component{
...
+            condition = false;
+            displayWebgold = ( <iframe src={'//webgold.'+process.env.DOMAIN+'/transactions'} style={ this.editIframeStyles }/>);
+            this.state.notDisplayCenter=true;
+        }
+
+        if (urlParams.create) {
+           condition = false;
+           nocomments = true;
+           this.state.notDisplayCenter=true;
+            displayCore =  ( <iframe src={'http://core.'+process.env.DOMAIN+'/create'} style={ this.editIframeStyles }/>);
```
Comment (AlexeyAnshakov) [inline WRIO-InternetOS/js/components/CreateDomCenter.js:197]: Remove http:

**Comment by AlexeyAnshakov** [comment]
Going to merge. Will remove http: by myself this time

**Pull request merged**